source/_components/life360.markdown

```diff
@@ -0,0 +1,234 @@
+---
+layout: page
+title: "Life360"
+description: "Instructions how to use Life360 to track devices in Home Assistant."
+date: 2019-04-24 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: life360.png
+ha_release: 0.93
+ha_category: Presence Detection
+ha_iot_class: Cloud Polling
+---
+
+The `life360` integration allows you to detect presence using the [unofficial API](#disclaimer) of [Life360](https://www.life360.com/). It can also automatically create Home Assistant zones based on Life360 Places.
+
+## {% linkable_title Life360 Account %}
+
+You must first [create a Life360 account](https://www.life360.com/websignup). Then add the following section to your `configuration.yaml` file:
+
+{% raw %}
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: life360
+    username: LIFE360_USERNAME
+    password: LIFE360_PASSWORD
+    prefix: life360
```

Comment:
Only required items in the first example

Reply:
Although prefix is not _technically_ required, I think it would be beneficial for most users to specify a unique prefix like this. (And since many, if not most, users simply copy the first example...)

I had thought about specifying a default, but then it wasn't clear how one would specify no prefix. Would either of the following be intuitive?
```text
prefix: ''
```
or
```text
prefix: null
```
I'm all ears if you have a better suggestion.

Reply:
Ok, decided to make the default 'life360', and any of the above would work to specify no prefix. I'll make a comment to that effect as well.

Reply:
Seems to be a good decision 😉